src/components/CreateCampaign/CreateCampaign.tsx

```diff
@@ -0,0 +1,88 @@
+import { Grid, createStyles } from '@mantine/core'
+import useCreateCampaignContext from 'hooks/useCreateCampaignContext'
+import CustomStepper from './CampaignStepper'
+import CampaignSummary from './CampaignSummary'
+import StepOne from './StepOne/StepOne'
+import StepTwo from './StepTwo/StepTwo'
+import StepThree from './StepThree/StepThree'
+import StepFour from './StepFour/StepFour'
+
+const useStyles = createStyles((theme) => {
+  return {
+    // TODO: Think about the idea to add the common container styles in the theme
+    container: {
+      backgroundColor: theme.colors.mainBackground[theme.fn.primaryShade()],
+      borderRadius: theme.radius.md,
+      boxShadow: theme.shadows.xs
+      // padding: theme.spacing.lg
+    },
+    lightGray: {
+      color: theme.fn.lighten(
+        theme.colors.secondaryText[theme.fn.primaryShade()],
+        theme.other.shades.lighten.lighter
+      )
+    },
+    dropZone: {
+      backgroundColor: theme.colors.lightBackground[theme.fn.primaryShade()],
+      border: '1px solid',
+      borderRadius: theme.radius.sm,
+      borderColor: theme.colors.decorativeBorders[theme.fn.primaryShade()],
+      height: 112
+    },
+    decorativeBorder: {
+      width: '99%',
+      height: '99%',
+      border: '1px dashed',
+      borderRadius: theme.radius.sm
+    }
+  }
+})
+
+const Wizard = ({ step }: { step: number }) => {
+  switch (step) {
+    case 0:
+      return <StepOne />
+    case 1:
+      return <StepTwo />
+    case 2:
+      return <StepThree />
+    case 3:
+      return <StepFour />
+    default:
+      return null
+  }
+}
+
```

Comment:
This switch can be replaced by array of the steps elements and the Wizard function can return the item at that index. Shorter and in case more steps come in, we would add the step in the array.